CHANGELOG.md

```diff
@@ -48,6 +47,8 @@ Same changes as in `@mui/x-date-pickers@7.3.2`.
 
 - [charts] Add an overlay for "no data" or "loading" states (#12817) @alexfauquette
 - [charts] Fix typos in documentation, translations and errors (#12941) @JCQuintas
+  This affects an undocumented [x-charts gradient id](https://github.com/mui/mui-x/pull/12941/files/4bf4bffbc2799a01a96bc7458a17318cf41c1722#diff-26c31cc69d6f51110f89e339578ef9b3d4a3551f79077fff73f7babb81c5099f) that could have unintended side effects
```

Comment:
Should we add this under the breaking changes section?

Reply:
Technically a breaking change yes, but it is also very small. @LukasTy suggested just adding it to the changelog. Do you think this should be in a `### Breaking change` section?

Reply:
IMHO, it's such a small BC that flagging it as one might be a bit counter-intuitive and cause more confusion than necessary. 🙈

Reply:
Go ahead with what you feel better, IMHO, putting in the breaking changes section would make more sense, as we used to do the same for all other BCs.